Traceback (most recent call last):
  File "/home/mis/Bureau/Scheduling Problem/./myenv/bin/rc2.py", line 1741, in <module>
    formula = WCNFPlus(from_file=files[0])
  File "/home/mis/Bureau/Scheduling Problem/myenv/lib/python3.10/site-packages/pysat/formula.py", line 4922, in __init__
    super(WCNFPlus, self).__init__(from_file=from_file, from_fp=from_fp,
  File "/home/mis/Bureau/Scheduling Problem/myenv/lib/python3.10/site-packages/pysat/formula.py", line 3757, in __init__
    self.from_file(from_file, comment_lead, compressed_with='use_ext')
  File "/home/mis/Bureau/Scheduling Problem/myenv/lib/python3.10/site-packages/pysat/formula.py", line 3804, in from_file
    with FileObject(fname, mode='r', compression=compressed_with) as fobj:
  File "/home/mis/Bureau/Scheduling Problem/myenv/lib/python3.10/site-packages/pysat/_fileio.py", line 108, in __init__
    self.open(name, mode=mode, compression=compression)
  File "/home/mis/Bureau/Scheduling Problem/myenv/lib/python3.10/site-packages/pysat/_fileio.py", line 123, in open
    self.fp = open(name, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'instance/2023/11_session_file.wcnf'

real	0m0,098s
user	0m0,085s
sys	0m0,012s
